### DISCUSSION TITLE
Metadata of Whatsapp Messages

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1171] [Q&A]
Looking at the image file, we see that it is a whatsapp message. This file is inside an itunes backup.
We can see data in the metadata area, but the file size is zero, even if exporting it results in an empty file.
Question: Where did he get this metadata from?
tks

![test](https://user-images.githubusercontent.com/381594/174449173-1dc1747f-6ab2-406f-92a5-bcdcbaf1a6df.png)

**Comment by lfcnassif** [comment]
It is a message extracted from a WhatsApp database in the iTunes backup. It has no size because it isn't an actual file, but info extracted from a relational database after joins etc. Right click on that message and select " Go to parent chat position", it should get you to the position in the whole conversation where it is.
